app/controllers/concerns/authentication_and_sso_concerns.rb

```diff
@@ -130,9 +127,7 @@ def sso_cookie_sign_credential_used
   # Info for logging purposes related to SSO.
   def sso_logging_info
     {
-      sso_cookies_enabled: Settings.sso.cookie_enabled,
```

Comment:
the `Settings.sso` settings are rarely changed, so we can just look at the commit history in the devops repo.